erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -643,8 +712,7 @@ def get_bom_operations(doctype, txt, searchfield, start, page_len, filters):
 @frappe.whitelist()
 def get_item_details(item, project = None, skip_bom_info=False):
 	res = frappe.db.sql("""
-		select stock_uom, description, item_name, allow_alternative_item,
-			include_item_in_manufacturing
+		select stock_uom, description, item_name, allow_alternative_item
 		from `tabItem`
 		where disabled=0
 			and (end_of_life is null or end_of_life='0000-00-00' or end_of_life > %s)
```

Comment:
line 728
`filters = {"item": item, "is_default": 1, "docstatus": 1}`
Currently a draft bom is also fetched as default bom in Work Order.